cfme/tests/services/test_rest_services.py

```diff
@@ -1914,12 +1920,14 @@ def test_create_second_cart(self, request, appliance, cart):
             initialEstimate: 1/4h
             tags: service
         """
-        # This will fail somehow once BZ 1493788 is fixed.
-        # There can be one and only one shopping cart for the authenticated user.
-        second_cart = appliance.rest_api.collections.service_orders.action.create(name="cart2")
-        second_cart = second_cart[0]
-        request.addfinalizer(second_cart.action.delete)
-        assert second_cart.state == 'cart'
+        with pytest.raises(
+            APIException,
+            match=(
+                "Could not create service order - Validation failed:"
+                " ServiceOrder: State has already been taken"
+            ),
+        ):
+            appliance.rest_api.collections.service_orders.action.create(name="cart2")
 
```

Comment:
Optional:
We can use faux factory.

Reply:
But the cart will anyway not be created, so keeping it as is.